simulation_parameters/microbe_stage/biomes.json

```diff
@@ -265,7 +265,7 @@
           "Meshes": [
             {
               "ScenePath": "res://assets/models/organelles/Oxytoxy.tscn",
-              "ConvexShapePath": "res://assets/models/organelles/Oxytoxy.shape"
+              "ConvexShapePath": "res://assets/models/organelles/ToxinVacuole.shape"
```

Comment:
So it looks like you renamed this one file, but if I'm looking at things correctly you didn't delete the `Oxytoxy.shape` file?